tox_conda/plugin.py

```diff
@@ -1,332 +1,525 @@
+                    " print(sys.version.split(os.linesep)[0]);"
+                    "print(sys.maxsize > 2**32);print(platform.system())"
+                ),
+            ],
+            "_get_python",
+        )
+        output = output.split(os.linesep)
 
-    # Check for venv.envconfig.sitepackages and venv.config.alwayscopy here
-    envdir = venv.envconfig.envdir
-    python = get_py_version(venv.envconfig, action)
+        implementation, version_info, version, is_64, platform_name = output
 
-    if venv.envconfig.conda_env is not None:
-        env_path = Path(venv.envconfig.conda_env)
+        is_64 = bool(is_64)
+        match = re.match(
+            (
+                r"sys\.version_info\(major=(\d+), minor=(\d+), micro=(\d+), releaselevel='(\w+)',"
+                r" serial=(\d+)\)"
+            ),
+            version_info,
+        )
+        version_info = VersionInfo(
+            major=int(match.group(1)),
+            minor=int(match.group(2)),
+            micro=int(match.group(3)),
+            releaselevel=match.group(4),
+            serial=int(match.group(5)),
+        )
+        extra = {"executable_path": exe_path}
+
+        return PythonInfo(implementation, version_info, version, is_64, platform_name, extra)
+
+    @property
+    def _package_tox_env_type(self) -> str:
+        return "virtualenv-pep-517"
+
+    @property
+    def _external_pkg_tox_env_type(self) -> str:
+        return "virtualenv-cmd-builder"
+
+    def _get_python_env_version(self):
+        # Try to use base_python config
+        match = re.match(r"python(\d)(?:\.(\d+))?(?:\.?(\d))?", self.conf["base_python"][0])
+        if match:
+            groups = match.groups()
+            version = groups[0]
+            if groups[1]:
+                version += ".{}".format(groups[1])
+            if groups[2]:
+                version += ".{}".format(groups[2])
+            return version
+        else:
+            return self.base_python.version_dot
+
+    @property
+    def runs_on_platform(self) -> str:
+        return sys.platform
+
+    def python_cache(self) -> Dict[str, Any]:
+        conda_dict = {}
+
+        conda_name = getattr(self.options, "conda_name", None)
+        if not conda_name:
+            conda_name = self.conf["conda_name"]
+
+        if conda_name:
+            conda_dict["env_spec"] = "-n"
+            conda_dict["env"] = conda_name
+        elif self.conf["conda_env"]:
+            conda_dict["env_spec"] = "-p"
+            conda_dict["env"] = str(self.env_dir)
+            env_path = Path(self.conf["conda_env"]).resolve()
+            conda_dict["env_path"] = str(env_path)
+            conda_dict["env_hash"] = hash_file(Path(self.conf["conda_env"]).resolve())
+        else:
+            conda_dict["env_spec"] = "-p"
+            conda_dict["env"] = str(self.env_dir)
+
+        _, conda_deps = self.conf["conda_deps"].unroll()
+        if conda_deps:
+            conda_dict["deps"] = conda_deps
```

Comment:
Why do we need deps for Python?

Reply:
Shouldn't these be already covered by deps durings the setup, the same way the tox uses it?

Reply:
Probably you are right, I am still learning how tox does the job internally. Can you point me to the tox code to be sure that I am checking the right approach.